app/views/layouts/application.html.slim

```diff
@@ -35,6 +35,8 @@ html lang="#{I18n.locale}"
 
     - if Figaro.env.google_analytics_key.present?
       = render 'shared/google_analytics/page_tracking'
+    - if Figaro.env.newrelic_browser_key.present?
```

Comment:
Is the `newrelic_browser_key` different from the existing `newrelic_license_key`?